examples/quickstart_etl/README.md

```diff
@@ -0,0 +1,247 @@
+
+<p align="center">
+    <img height="500" src="../../docs/next/public/images/quickstarts/basic/step-2-7-view-assets.png" />
+</p>
+
+This metadata would be useful for monitoring and maintaining the asset as you iterate. Similarly, we've also recorded some metadata in the `hackernews_topstories` asset. You can filter the compute logs by typing the asset name (e.g. `hackernews_topstories`) or the event type (e.g. `type:ASSET_MATERIALIZATION`) in the "Log Filter" input box:
+
+<p align="center">
+    <img height="500" src="../../docs/next/public/images/quickstarts/basic/step-2-8-filter.png" />
+</p>
+
+Then, you can find the `hackernews_topstories` asset has two metadata entries: `num_records` and `preview`. Both are defined [in `quickstart_etl/assets/hackernews.py`](./quickstart_etl/assets/hackernews.py), in which we record the first five rows of the output Pandas DataFrame in the `preview` metadata entry as Markdown type -- This could help debug and keep your assets easily monitored. Click "Show Markdown", and you will see the preview of the output data frame:
+
+<p align="center">
+    <img height="500" src="../../docs/next/public/images/quickstarts/basic/step-2-9-preview.png" />
+</p>
+
+Note: You'll find a `path` metadata attached to every asset. This is because by default assets are materialized to pickle files on your local filesystem. In most projects, your assets will be materialized to a production system, and you can fully customize the I/O using [I/O managers](https://docs.dagster.io/concepts/io-management/io-managers).
+
+## Step 3: Scheduling a daily job
+
+Finally, let's refresh our plots every day, so we can monitor the popular topics over time. To do so, we can use [Schedules](https://docs.dagster.io/concepts/partitions-schedules-sensors/schedules#schedules).
+
+We've defined a daily schedule in [quickstart_etl/repository.py](./quickstart_etl/repository.py) (which is Dagster's word for "project") for all assets that are defined in the [quickstart_etl/assets/](./quickstart_etl/assets) module.
+
+Now, let's set up the daily job with Dagster.
+
+In the UI, clicking "all_assets_job" in the left nav will bring you to the job definition page. In the left nav, it indicates the `all_assets_job` has a schedule associated with it; on the job page, you will find more info about the schedule.
+
+<p align="center">
+    <img height="500" src="../../docs/next/public/images/quickstarts/basic/step-3-1-schedule-off.png" />
+</p>
+
+ You can simply turn on the schedule switch to set up the daily job we defined in [quickstart_etl/repository.py](./quickstart_etl/repository.py).
```

Comment:
```suggestion
 Turn on the schedule switch to set up the daily job we defined in [`quickstart_etl/repository.py`](./quickstart_etl/repository.py):
```

Nit - I'm personally not a fan of using words like `simply` to describe actions in tech docs/guides. What we consider to be simple may not be by someone else, and it also adds fluff. 🤷‍♀️